modules/drivers/bigquery-cloud-sdk/test/metabase/driver/bigquery_cloud_sdk_test.clj

```diff
@@ -269,21 +270,18 @@
                                 PARTITION BY
                                 transaction_date
                                 OPTIONS (
-                                partition_expiration_days = 3,
```

Comment:
Can you explain why these were added before and now removed?

Reply:
I added it because it's from the documentation code, but they're irrelevant to the tests. Don't want to give people the wrong idea about what's being tested.

Reply:
Makes sense, just checking